src/common/mmo.h

```diff
@@ -256,6 +256,12 @@
 #define MAX_ELESKILLTREE 3
 #endif
 
+#define MAX_ITEM_OPTIONS 5 // Maximum item options [Smokexyz]
```

Comment:
Please add #ifndef MAX_ITEM_OPTIONS before 259 line
and #endif after 263 line
This will allow override this settings from build script without changing code.